### ISSUE TITLE
Crashes when loading a schematic, cause indeterminate

### ISSUE THREAD

**Issue by akirapink** [sakura-ryoko/litematica#96]
it might be that this schematic was made with Obj2Schematic, but i do not know
it might be the other mods, but i do not know
log output:
https://mclo.gs/u6Ecxeq

**Comment by sakura-ryoko** [comment]
That error is normally due to a malformed file; such as from an invalid conversion, or an empty file.

**Comment by K4n3k11** [comment]
for me the mod doesnt work at all. i'm trying to load schematics but the game crashes

**Comment by sakura-ryoko** [comment]
There is another mod doing something to the Litematica menu screens.  Take a look at this mod:

```
[14:53:19] [Render thread/INFO]: [FANCYMENU] ScreenCustomizationLayer registered: fi.dy.masa.litematica.gui.GuiMainMenu
[14:53:22] [Render thread/INFO]: [FANCYMENU] ScreenCustomizationLayer registered: fi.dy.masa.litematica.gui.GuiSchematicLoad
```

**Comment by K4n3k11** [comment]
sorry to disturb you again but it's not other mods causing it to crash, but it's the mod itself. I tried removing every mod except for the litematica, but it still crashes it, i can give you the crash log if it's needed

**Comment by sakura-ryoko** [comment]
Well if your not the original poster, than please post your crash log.

**Comment by K4n3k11** [comment]
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2024-11-02 18:58:23
Description: mouseClicked event handler

java.lang.RuntimeException: Mixin transformation of fi.dy.masa.litematica.schematic.placement.SchematicPlacement failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at knot//fi.dy.masa.litematica.gui.GuiSchematicLoad$ButtonListener.actionPerformedWithButton(GuiSchematicLoad.java:187)
	at knot//fi.dy.masa.malilib.gui.button.ButtonBase.onMouseClickedImpl(ButtonBase.java:85)
	at knot//fi.dy.masa.malilib.gui.widgets.WidgetBase.onMouseClicked(WidgetBase.java:92)
	at knot//fi.dy.masa.malilib.gui.GuiBase.onMouseClicked(GuiBase.java:293)
	at knot//fi.dy.masa.malilib.gui.GuiListBase.onMouseClicked(GuiListBase.java:92)
	at knot//fi.dy.masa.malilib.gui.GuiBase.method_25402(GuiBase.java:235)
	at knot//net.minecraft.class_312.mixinextras$bridge$method_25402$47(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$zcn000$architectury$onGuiMouseClicked(class_312.java:1085)
	at knot//net.minecraft.class_312.mixinextras$bridge$wrapOperation$zcn000$architectury$onGuiMouseClicked$50(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$bdj000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:1545)
	at knot//net.minecraft.class_312.method_1601(class_312.java:114)
	at knot//net.minecraft.class_312.method_22686(class_312.java:215)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:215)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3438)
	at knot//com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:149)
	at knot//com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:170)
	at knot//net.minecraft.class_1041.method_15998(class_1041.java:314)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1370)
	at knot//net.minecraft.class_310.method_1514(class_310.java:933)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 30 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [syncmatica.litematica_mixin.json:MixinSchematicPlacement from mod syncmatica] from phase [DEFAULT] in config [syncmatica.litematica_mixin.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379)
	... 33 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on setNull could not find any targets matching '<init>(Lfi/dy/masa/litematica/schematic/LitematicaSchematic;Lnet/minecraft/util/math/BlockPos;Ljava/lang/String;ZZLfi/dy/masa/litematica/schematic/placement/SchematicPlacementManager;)V' in fi/dy/masa/litematica/schematic/placement/SchematicPlacement. Using refmap syncmatica-fabric-1.21.3-refmap.json [INJECT_PREPARE Applicator Phase -> syncmatica.litematica_mixin.json:MixinSchematicPlacement from mod syncmatica -> Prepare Injections -> handler$bnh000$syncmatica$setNull(Lfi/dy/masa/litematica/schematic/LitematicaSchematic;Lnet/minecraft/class_2338;Ljava/lang/String;ZZLfi/dy/masa/litematica/schematic/placement/SchematicPlacementManager;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse ->  -> Validate Targets]
	at org.spongepowered.asm.mixin.injection.selectors.TargetSelectors.validate(TargetSelectors.java:346)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:369)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:340)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:331)
	at org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo.<init>(CallbackInjectionInfo.java:48)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:196)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:664)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1399)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:731)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:315)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:246)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:437)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:418)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 33 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at knot//fi.dy.masa.litematica.gui.GuiSchematicLoad$ButtonListener.actionPerformedWithButton(GuiSchematicLoad.java:187)
	at knot//fi.dy.masa.malilib.gui.button.ButtonBase.onMouseClickedImpl(ButtonBase.java:85)
	at knot//fi.dy.masa.malilib.gui.widgets.WidgetBase.onMouseClicked(WidgetBase.java:92)
	at knot//fi.dy.masa.malilib.gui.GuiBase.onMouseClicked(GuiBase.java:293)
	at knot//fi.dy.masa.malilib.gui.GuiListBase.onMouseClicked(GuiListBase.java:92)
	at knot//fi.dy.masa.malilib.gui.GuiBase.method_25402(GuiBase.java:235)
	at knot//net.minecraft.class_312.mixinextras$bridge$method_25402$47(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$zcn000$architectury$onGuiMouseClicked(class_312.java:1085)
	at knot//net.minecraft.class_312.mixinextras$bridge$wrapOperation$zcn000$architectury$onGuiMouseClicked$50(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$bdj000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:1545)
	at knot//net.minecraft.class_312.method_1601(class_312.java:114)
	at knot//net.minecraft.class_312.method_22686(class_312.java:215)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:215)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3438)
	at knot//com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:149)
	at knot//com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:170)

-- Affected screen --
Details:
	Screen name: fi.dy.masa.litematica.gui.GuiSchematicLoad
Stacktrace:
	at knot//net.minecraft.class_312.method_1601(class_312.java:114)
	at knot//net.minecraft.class_312.method_22686(class_312.java:215)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:215)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3438)
	at knot//com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:149)
	at knot//com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:170)
	at knot//net.minecraft.class_1041.method_15998(class_1041.java:314)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1370)
	at knot//net.minecraft.class_310.method_1514(class_310.java:933)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Mouse --
Details:
	Scaled X: 40.5
	Scaled Y: 226.5
	Button: 0
Stacktrace:
	at knot//net.minecraft.class_312.method_1601(class_312.java:114)
	at knot//net.minecraft.class_312.method_22686(class_312.java:215)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:215)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3438)
	at knot//com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:149)
	at knot//com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:170)
	at knot//net.minecraft.class_1041.method_15998(class_1041.java:314)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1370)
	at knot//net.minecraft.class_310.method_1514(class_310.java:933)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Uptime --
Details:
	JVM uptime: 26064.154s
	Wall uptime: 26055.580s
	High-res time: 26053.678s
	Client ticks: 520911 ticks / 26045.550s
Stacktrace:
	at knot//net.minecraft.class_310.method_1587(class_310.java:2443)
	at knot//net.minecraft.class_310.method_54580(class_310.java:1002)
	at knot//net.minecraft.class_310.method_1514(class_310.java:954)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 1 total; [class_746['K4n3k1_'/7, l='ClientLevel', x=239.84, y=185.00, z=-175.87]]
	Chunk stats: 1521, 1021
	Level dimension: minecraft:overworld
	Level spawn location: World: (496,64,-336), Section: (at 0,0,0 in 31,4,-21; chunk contains blocks 496,-64,-336 to 511,1023,-321), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,1023,-1)
	Level time: 177659598 game time, 202154040 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
	Tracked entity count: 157

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, chat_heads, cloth-config, entity_model_features, entity_texture_features, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, fallingleaves, gammautils, litematica, malilib, minihud, modmenu, reeses-sodium-options, sodium, sound_physics_remastered, status-effect-bars, syncmatica, tweakeroo, voicechat, wi_zoom, yet_another_config_lib_v3

-- System Details --
Details:
	Minecraft Version: 1.21.3
	Minecraft Version ID: 1.21.3
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 876158712 bytes (835 MiB) / 5536481280 bytes (5280 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 6
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 13
	Microarchitecture: Coffee Lake
	Frequency (GHz): 3.70
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 6
	Graphics card #0 name: NVIDIA GeForce RTX 2080 SUPER
	Graphics card #0 vendor: NVIDIA
	Graphics card #0 VRAM (MiB): 8192.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 32.0.15.6603
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 25658.29
	Virtual memory used (MiB): 21983.60
	Swap memory total (MiB): 9338.95
	Swap memory used (MiB): 962.19
	Space in storage for jna.tmpdir (MiB): available: 482122.06, total: 952516.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 482122.06, total: 952516.00
	Space in storage for io.netty.native.workdir (MiB): available: 482122.06, total: 952516.00
	Space in storage for java.io.tmpdir (MiB): available: 482122.06, total: 952516.00
	Space in storage for workdir (MiB): available: 482122.06, total: 952516.00
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		architectury: Architectury 14.0.4
		audioplayer: AudioPlayer 1.21.3-1.13.0
			admiral: Admiral 0.4.6+1.21.1
		chat_heads: Chat Heads 0.13.5
		cloth-config: Cloth Config v16 16.0.141
			cloth-basic-math: cloth-basic-math 0.6.1
		entity_model_features: Entity Model Features 2.2.7
		entity_texture_features: Entity Texture Features 6.2.6
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		fabric-api: Fabric API 0.107.0+1.21.3
			fabric-api-base: Fabric API Base 0.4.49+d21566ae40
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.79+c47b9d4340
			fabric-biome-api-v1: Fabric Biome API (v1) 14.0.6+6c3b5d4940
			fabric-block-api-v1: Fabric Block API (v1) 1.0.28+c47b9d4340
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.16+c47b9d4340
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.3+c47b9d4340
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.23+c47b9d4340
			fabric-command-api-v1: Fabric Command API (v1) 1.2.57+f71b366f40
			fabric-command-api-v2: Fabric Command API (v2) 2.2.36+c47b9d4340
			fabric-commands-v0: Fabric Commands (v0) 0.2.74+df3654b340
			fabric-content-registries-v0: Fabric Content Registries (v0) 9.1.6+c47b9d4340
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.1+7f945d5b40
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.9.1+1ff58ec240
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.3+c47b9d4340
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.2.0+d9873d8b40
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 21.0.15+c47b9d4340
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.7+c47b9d4340
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.0.6+c47b9d4340
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 2.0.10+c47b9d4340
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.59+c47b9d4340
			fabric-item-api-v1: Fabric Item API (v1) 11.1.6+c47b9d4340
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.16+e7c9ab6640
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.53+c47b9d4340
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.51+df3654b340
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.4.0+4402f4ee40
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.25+3f89f5a540
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.13+c47b9d4340
			fabric-message-api-v1: Fabric Message API (v1) 6.0.20+c47b9d4340
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 3.0.8+c47b9d4340
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.4+89cb0a4e40
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 17.0.8+c47b9d4340
			fabric-particles-v1: Fabric Particles (v1) 4.0.9+c47b9d4340
			fabric-recipe-api-v1: Fabric Recipe API (v1) 7.0.3+c47b9d4340
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.2.1+89cb0a4e40
			fabric-renderer-api-v1: Fabric Renderer API (v1) 4.0.4+c47b9d4340
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.9.4+c47b9d4340
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.54+73761d2e40
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.14+c47b9d4340
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.82+df3654b340
			fabric-rendering-v1: Fabric Rendering (v1) 8.0.6+c47b9d4340
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.8+c47b9d4340
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.0.6+e82f21f740
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.33+c47b9d4340
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.99+c47b9d4340
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.29+c47b9d4340
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.1+c47b9d4340
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.8+c47b9d4340
		fabricloader: Fabric Loader 0.16.9
			mixinextras: MixinExtras 0.4.1
		fallingleaves: Falling Leaves 1.16.3
		gammautils: Gamma Utils 2.2.1
		java: OpenJDK 64-Bit Server VM 21
		litematica: Litematica 0.20.0-sakura.3
		malilib: MaLiLib 0.22.0-sakura.3
		minecraft: Minecraft 1.21.3
		minihud: MiniHUD 0.33.0-sakura.4
		modmenu: Mod Menu 12.0.0-beta.1
			placeholder-api: Placeholder API 2.4.0-pre.2+1.21
		mousetweaks: Mouse Tweaks 2.27
		reeses-sodium-options: Reese's Sodium Options 1.8.0-beta.5+mc1.21.3
		sodium: Sodium 0.6.0-beta.4+mc1.21.3
		sound_physics_remastered: Sound Physics Remastered 1.21.3-1.4.6
		status-effect-bars: Status Effect Bars 1.0.7
		syncmatica: Syncmatica 0.3.12-sakura.9
		tweakeroo: Tweakeroo 0.22.0-sakura.3
		voicechat: Simple Voice Chat 1.21.3-2.5.24
		wi_zoom: WI Zoom 1.5-MC1.21.3
		yet_another_config_lib_v3: YetAnotherConfigLib 3.6.1+1.21.2-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Launched Version: fabric-loader-0.16.9-1.21.3
	Launcher name: minecraft-launcher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: NVIDIA GeForce RTX 2080 SUPER/PCIe/SSE2 GL version 3.2.0 NVIDIA 566.03, NVIDIA Corporation
	Window size: 854x480
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated.' x 1
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Render Distance: 16/16 chunks
	Resource Packs: vanilla, fabric, chat_heads, cloth-config, entity_model_features, entity_texture_features, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, fallingleaves, gammautils, litematica, malilib, minihud, modmenu, reeses-sodium-options, sodium, sound_physics_remastered, status-effect-bars, syncmatica, tweakeroo (incompatible), voicechat, wi_zoom, yet_another_config_lib_v3 (incompatible)
	Current Language: en_us
	Locale: it_IT
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 6x Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz
	Server Running: true
	Player Count: 1 / 8; [class_3222['K4n3k1_'/7, l='ServerLevel[MISTERSES SERVER]', x=239.84, y=185.00, z=-175.87]]
	Active Data Packs: vanilla, fabric, audioplayer, fabric-convention-tags-v2, file/1024block-heigth-1-21.zip (incompatible)
	Available Data Packs: audioplayer, fabric, fabric-convention-tags-v2, file/1024block-heigth-1-21.zip (incompatible), minecart_improvements, redstone_experiments, trade_rebalance, vanilla, winter_drop
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	World Seed: 1381934048578206924
	Suppressed Exceptions: ~~NONE~~

**Comment by sakura-ryoko** [comment]
Your Crash is Syncmatica related.  Download the fix from my Syncmatica Repo.

https://github.com/sakura-ryoko/syncmatica/releases/tag/1.21.3-sakura.10
